### PR TITLE
Switch to FileHandler

### LINKED ISSUE #960: Remove copyfile here so that we can get rid of solo dependency
https://github.com/NOAA-EMC/GDASApp/blob/a3c3c10a62ac54dde502598129f8b543197325a6/ush/socaincr2mom6.py#L29

### PR DESCRIPTION
**Pull request by guillaumevernieres**
- fixes #960 
I think?

### CODE REVIEW

File: ush/socaincr2mom6.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 from netCDF4 import Dataset
 from scipy.interpolate import griddata
 import ufsda
-from wxflow import YAMLFile
+from wxflow import (FileHandler, YAMLFile)
```
Comment (CoryMartin-NOAA) [inline ush/socaincr2mom6.py]: Why in parentheses?

**Review by CoryMartin-NOAA — DISMISSED** [review]
Looks good, thanks for doing this so quickly! Agile development at its finest!

File: ush/socaincr2mom6.py (hunk 1 above)
Comment (guillaumevernieres) [reply ush/socaincr2mom6.py]: I saw @AndrewEichmann-NOAA doing this and I thought it looked cool ... And since I want to be cool too ...

**Comment by guillaumevernieres** [comment]
> Looks good, thanks for doing this so quickly! Agile development at its finest!

damn ... You probably just jinxed the ci test @CoryMartin-NOAA !!!! Your fault if the tests fail.

**Comment by guillaumevernieres** [comment]
Was there a reason for the `()` for the import @AndrewEichmann-NOAA ? I vaguely remember this being mentioned a while back.

**Comment by CoryMartin-NOAA** [comment]
@guillaumevernieres a google suggests it's for line continuation, but it's a short enough line

**Review by CoryMartin-NOAA — APPROVED** [review]

**Pull request merged**